### PR TITLE
Update stale.yml

### PR DESCRIPTION
**Pull request by frederikprijck**
A small update to the comment in the stale.yml file, as it had me confused

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ daysUntilStale: 30
 # Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
 daysUntilClose: 7
 
-# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
```
Comment (stevehobbsdev) [inline .github/stale.yml:9]: I would reword this a little. Just a suggestion:

```suggestion
# Only issues or pull requests with all of these labels are considered by StaleBot. Defaults to `[]` (disabled)
```

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]

File: .github/stale.yml (hunk 1 above)
Comment (frederikprijck) [reply .github/stale.yml:9]: Ok. I took the comment straight from the stalebot docs. But can reword if u prefer.

Comment (stevehobbsdev) [reply .github/stale.yml:9]: Really? It was more just to remove the typo but I don't think it reads right anyway 🤔

Comment (frederikprijck) [reply .github/stale.yml:9]: https://github.com/probot/stale 
I will take your changes, no problem!

**Review by stevehobbsdev — APPROVED** [review]
